common/python/schema_validation/data/sdk_inData.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
inData and outData structure is same. same comment as https://github.com/hyperledger/avalon/pull/600/files#r465467812